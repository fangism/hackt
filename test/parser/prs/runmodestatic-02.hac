// "runmodestatic-02.hac"

bool x, y;
prs {
	x	=> y-
}
spec {
	runmodestatic(x, y)	// wrong number of args
}

