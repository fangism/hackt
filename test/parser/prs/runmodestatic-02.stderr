The 'runmodestatic' directive requires exactly 1 arguments.
	at [8:15..20]
ERROR: at least one error in spec body.
